server.js

```diff
@@ -1067,6 +1067,43 @@ cache(function(data, match, sendBadge, request) {
   }
 }));
 
+// LGTM alerts integration
+camp.route(/^\/lgtm\/alerts\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var projectId = match[1]; // eg, `g/apache/cloudstack`
+  var format = match[2];
+  var url = 'https://lgtm.com/api/v0.1/project/' + projectId + '/details';
+  var badgeData = getBadgeData('lgtm', data);
```

Comment:
Could you use the ES6 style `let` and `const` (I know we have a lot of code using `var`, but we're trying to update as we add new code).